Develop

Rupesh-Darimisetti opened the pull request.

Rupesh-Darimisetti committed:
fix errors in file

Rupesh-Darimisetti committed:
Merge branch 'master' of https://github.com/Rupesh-Darimisetti/dart-sdk into develop

irsayvid committed:
format json encode

irsayvid committed:
Merge branch 'develop' of github-rupesh:Rupesh-Darimisetti/dart-sdk into develop

mraleph commented:
I kindly ask you to stop sending meaningless changes. 

Your change to `PRESUBMIT.py` breaks it - which you would have known if you followed proper contribution workflow and not taken a shortcut of sending a PR on GitHub (which are left open as a courtesy and not as a main way to develop Dart SDK). Changing code in legacy newsletters archive also does not make much sense. 

If you wish to make some contributions on consider instead *asking* first (eg look at the issue tracker)  - there might be places where help is wanted, rather then just doing changes that do not make sense.

Rupesh-Darimisetti commented:
@googlebot I fixed it.

On 29-May-2021 22:05, "google-cla[bot]" ***@***.***> wrote:

> We found a Contributor License Agreement for you (the sender of this pull
> request), but were unable to find agreements for all the commit author(s)
> or Co-authors. If you authored these, maybe you used a different email
> address in the git commits than was used to sign the CLA (login here
> <https://cla.developers.google.com/> to double check)? If these were
> authored by someone else, then they will need to sign a CLA as well, and
> confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment @googlebot
> I fixed it.. If the bot doesn't comment, it means it doesn't think
> anything has changed.
>
> ℹ️ *Googlers: Go here
> <https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fdart-lang%2Fsdk%2Fpull%2F46185>
> for more info*.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dart-lang/sdk/pull/46185#issuecomment-850861445>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AL2KFMND7KK7LWJ4TRAVTNTTQEJWBANCNFSM45YOKABA>
> .
>
